content/client-lib-development-guide/features.textile

```diff
@@ -668,8 +668,8 @@ h3(#realtime-presence). Presence
 ** @(RTP16b)@ If the channel is @ATTACHING@ or @INITIALIZED@, then if @ClientOptions.queueMessages@ has not been explicitly set to @false@, the presence messages should be queued at a channel level, to be handled per @RTP16a@ once the channel becomes @ATTACHED@, or failed if the channel state becomes @SUSPENDED@, @FAILED@, or @DETACHED@ first
 ** @(RTP16c)@ In any other case the operation should result in an error
 * @(RTP6)@ @Presence#subscribe@ function:
-** @(RTP6a)@ Subscribe with no arguments subscribes a listener to all presence messages
-** @(RTP6b)@ Subscribe with a single action argument - such as @ENTER@, @LEAVE@, @UPDATE@ or @PRESENT@ - subscribes a listener to receive only presence messages with that action
+** @(RTP6a)@ Subscribe with a single listener argument subscribes a listener to all presence messages
+** @(RTP6b)@ Subscribe with a listener argument and an action argument - such as @ENTER@, @LEAVE@, @UPDATE@ or @PRESENT@ - subscribes a listener to receive only presence messages with that action
```

Comment:
The listener would be after the action argument in the code so it makes sense to write them that way here

```suggestion
** @(RTP6b)@ Subscribe with an action argument and a listener argument - such as @ENTER@, @LEAVE@, @UPDATE@ or @PRESENT@ - subscribes a listener to receive only presence messages with that action
```

Reply:
Thanks, done!